src/libertem/api.py

```diff
@@ -66,7 +66,7 @@ def load(self, filetype: str, *args, **kwargs) -> DataSet:
 
     load.__doc__ = load.__doc__ % {'types': ", ".join(filetypes.keys())}
 
-    def create_mask_job(self, factories, dataset):
+    def create_mask_job(self, factories, dataset, use_sparse=None):
```

Comment:
As discussed, add `use_sparse` to docstring.

Reply:
Done! :-)